### DISCUSSION TITLE
Migrazione Home Assistant OS da Rpi 3 a Rpi 4

### DISCUSSION THREAD

**Discussion by papinist** [indomus/forum#563] [HUB: Home Assistant]
Ciao a tutti,
nella mia 'sistemazione' di Home Assistant da progetto di test a sistema effettivo per il controllo della casa, dopo aver migrato da docker a HA OS su Rpi 3, ora vorrei passare da Rpi 3 a Rpi 4 in modo da avere, in futuro, maggiore flessibilità e potenza (magari per motion eye o simili).

Penso che con un semplice snapshot la migrazione sia indolore, quello che però mi preoccupa è la chiavetta zigbee inserita su una usb: quando la sposterò nel nuovo sistema non sono sicuro che verrà riconosciuta con lo stesso indirizzo, che poi deve essere indicato su zigbee2mqtt.
Come posso verificare l'indirizzo che le assegna il sistema?

Vedete altre problematiche in questo tipo di migrazione?

**Comment by papinist** [comment]
Mi rispondo da solo: cercando nella community internazionale ho letto che va bene lo snapshot ma non va ripristinato il 'HA core'. Così ho fatto ed ha funzionato tutto. Fortunatamente (o "by design", non lo so) la chiavetta zigbee è ripartita da sola!

In seguito ho fatto anche il passaggio da SD a SSD M.2 seguendo altre guide senza nessun problema.

**Reply by ganzerlimino** [reply]
Wow, stavo giusto cercando traccia di qualcuno che l'avesse fatto, mille grazie per la tua testimonianza!

**Reply by papinist** [reply]
Figurati, lieto di essere stato di aiuto :)
